### DISCUSSION TITLE
Welcome to FreeKill Discussions!

### DISCUSSION THREAD

**Discussion by Notify-ctrl** [Qsgs-Fans/FreeKill#68] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jiarenrengyuan** [comment]
![1481422558cec1b6821bacf9d46fe9a](https://github.com/Qsgs-Fans/FreeKill/assets/144088666/3612d15a-2939-448c-9817-e5029ddc5ac4)
我在运行程序的时候会有一个DOS窗口...这算不算是一个小问题...?

**Reply by Notify-ctrl** [reply]
不算问题，他就是得有个窗口在这里
